### PR TITLE
anirban fix: checks

### LINKED ISSUE #58: Fix lintr errrors
### Guidelines

- [X] I agree to follow this project's Contributing Guidelines.

### Project Version

_No response_

### Platform and OS Version

_No response_

### Existing Issues

_No response_

### What happened?

There are lintr errors preventing the CI to pass

### Steps to reproduce

run lintr::lintr_package()

### Expected behavior

Fix lintr errors

### Attachments

_No response_

### Screenshots or Videos

_No response_

### Additional Information

_No response_

### LINKED ISSUE #60: fix lints and bioc checks

### PR DESCRIPTION
**Pull request by anirbanshaw24**
Closes #58 

*Have you read the [Contributing Guidelines](https://github.com/Appsilon/.github/blob/main/CONTRIBUTING.md)?*

Issue #

## Changes description

*Clearly and concisely describe what's in this pull request. Include screenshots, if necessary.*

### CODE REVIEW

**Review by federiva — APPROVED** [review]
LGTM!

**Comment by federiva** [comment]
It might be a good idea to add some **snapshot tests** to the package

File: .github/workflows/r.yml [hunk 1]
```diff
@@ -42,6 +43,8 @@ jobs:
         if: always()
         shell: Rscript {0}
         run: |
+          remotes::install_cran("devtools")
```
Comment (federiva) [inline .github/workflows/r.yml:46]: A comment: We could extract this to its own R script, but I guess that it's not a priority right now

Comment (anirbanshaw24) [reply .github/workflows/r.yml:46]: Thanks. Yes, good idea! No time right now but I will add it for the next version backlog.

**Pull request merged**